test/nn/models/test_explainer.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+import torch
+
+from torch_geometric.nn import GAT, GCN, to_captum
+
+GCN = GCN(3, 16, 2, 7, dropout=0.5)
+GAT = GAT(3, 16, 2, 7, heads=2, concat=False)
+
+mask_type = ['edge', 'node_and_edge', 'node']
+
+
+@pytest.mark.parametrize('mask_type', mask_type)
+@pytest.mark.parametrize('model', [GCN, GAT])
+@pytest.mark.parametrize('node_idx', [None, 1])
+def test_to_captum(model, mask_type, node_idx):
```

Comment:
We do not really test `captum` support here. We could add `captum` dependency to `full_requires`, and only perform this test in case `captum` is installed (via `pytest.mark.skipif`).

Reply:
Okay, I'll add that! I wasn't sure if you wanna have Captum as a dependency.

Reply:
I added a test for most of the [attribution methods provided in Captum](https://captum.ai/api/attribution.html). FeaturePermutation, GuidedGradCam, LRP, DeepLift, and DeepLiftShap can not be applied. They either require several samples or modify the input by adding more samples. This doesn't work with our edge_mask approach, at the moment.